tests/fib/test_fib.py

```diff
@@ -158,6 +161,26 @@ def setup_hash(self, testbed, duthost, ptfhost):
         in_ports_name = get_router_interface_ports(config_facts, testbed)
         g_vars['in_ports'] = [config_facts.get('port_index_map', {})[p] for p in in_ports_name]
 
+        # add some vlan for hash_key vlan-id test
```

Comment:
this setup needs to move to a separate fixture, for example, setup_vlan. Then, only test_hash_key_vlan_id load this fixture.

Reply:
> this setup needs to move to a separate fixture, for example, setup_vlan. Then, only test_hash_key_vlan_id load this fixture.

Add a determination of whether vlan-id is in hash_keys.

Move setup_vlan to a separate fixture for only test_hash_key_vlan_id load this fixture. That means break each hash field into one test, should we change it in this PR?

Reply:
it is better to separate each hash field into one test. you can do it in future prs.